src/platforms/native/guides/wasm/index.mdx

```diff
@@ -12,8 +12,9 @@ Support for it in Sentry is limited by what's possible in that ecosystem.
 
 <Note>
 
-Want to be an early adopter, but running into issues? Please reach out to
-us using our [Forum](https://forum.sentry.io/), we would love to talk.
+Want to be an early adopter, but running into issues? Please reach out to us
+[on GitHub](https://github.com/getsentry/symbolicator/issues/new/choose), we
+would love to talk.
```

Comment:
Is Symbolicator the right place to send WASM traffic? cc: @getsentry/owners-native

Reply:
Probably: https://github.com/getsentry/sentry-native

Reply:
I considered that, but the other two GitHub links in this file are to Symbolicator.

Thoughts @Swatinem?

Reply:
I'm going to stick with Symbolicator based on the other links.